pages/api/ai/decisionMatrix.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios'
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+import { matrixToken } from '../../../utils/constants/global'
+
+const handler = async (req: NextApiRequest, res: NextApiResponse<any>) => {
+    const { decision, context } = req.query
+    if (!decision) {
+        return res.status(400).json({ err: 'Missing decision' })
+    }
+    try {
+        const headers = {
+            'Content-Type': 'application/json',
+        }
+        const url = 'http://3.130.66.102:5001/api/v1/decision_table'
```

Comment:
This still is hard coded. Please store http://3.130.66.102:5001/  as an env var.